### PR TITLE
Add Open-Source GitGat to OPA list

### PR DESCRIPTION
**Pull request by MrBLightning**
Signed-off-by: Barak Brudo [barak@scribesecurity.com](mailto:barak@scribesecurity.com)

GitGat is an open-source project written with OPA (in Rego) by Scribe Security. It's designed to evaluate your SCM platform (GitHub) and give a security posture report with actionable recommendations.

### CODE REVIEW

**Comment by srenatus** [comment]
Thanks! Please check the DCO requirements, see [here](https://github.com/open-policy-agent/opa/pull/5060/checks?check_run_id=8067319056)

**Comment by anderseknert** [comment]
While it’s optional, perhaps you’d want to include a logo? 😊

File: docs/website/data/integrations.yaml [hunk 1]
```diff
@@ -1454,6 +1454,21 @@ integrations:
       - https://github.com/StyraInc/graphql-apollo-example
     tutorials:
       - https://www.openpolicyagent.org/docs/graphql-api-authorization/
+      
+gitgat:
```
Comment (srenatus) [inline docs/website/data/integrations.yaml:1458]: Indentation is wrong here, this doesn't get picked up by the website build logic. Plase indent this like graphql above.

**Comment by MrBLightning** [comment]
@anderseknert, I saw you suggested I add a logo. I'd gladly do that but I didn't see a place for it anywhere in the file. If you could direct me to where it should be added I'll add the link to it. If there is anything else I missed I'll gladly correct it.

**Comment by anderseknert** [comment]
> @anderseknert, I saw you suggested I add a logo. I'd gladly do that but I didn't see a place for it anywhere in the file. If you could direct me to where it should be added I'll add the link to it. If there is anything else I missed I'll gladly correct it.

If you check the comment at the top of the integrations.yaml file, you'll find a link to instructions for that 🙂

**Comment by MrBLightning** [comment]
> > @anderseknert, I saw you suggested I add a logo. I'd gladly do that but I didn't see a place for it anywhere in the file. If you could direct me to where it should be added I'll add the link to it. If there is anything else I missed I'll gladly correct it.
> 
> If you check the comment at the top of the integrations.yaml file, you'll find a link to instructions for that 🙂

Thanks. Added a logo. 
I hope it's all ok now :)

**Comment by anderseknert** [comment]
@MrBLightning almost there :) you'll need to rename the logo to match the integration, i.e `gitgat.png` with lowercase letters. Also, I don't think you've addressed @srenatus comment above?

**Comment by MrBLightning** [comment]
Signed-off-by: Barak Brudo [barak@scribesecurity.com](mailto:barak@scribesecurity.com)
Did the rebasing as suggested in the DCO requirements.
I have no idea if it worked. Did I miss anything?

File: docs/website/data/integrations.yaml (hunk 1 above)
Comment (anderseknert) [reply docs/website/data/integrations.yaml:1458]: @MrBLightning

**Comment by srenatus** [comment]
I'd really like to get this over with 😅 

Please:
- [ ] rebase to fix the conflict
- [ ] sort out DCO
- [ ] fix the indentation in integrations.yaml

**Comment by srenatus** [comment]
Please re-open when you find the time.

**Pull request closed without merging**